_studio/shared/umc/codec/h264_dec/include/umc_h264_frame_info.h

```diff
@@ -116,11 +116,10 @@ class H264DecoderFrameInfo : public H264DecoderLayer
 {
 public:
 
-    H264DecoderFrameInfo(H264DecoderFrame * pFrame,  H264_Heap_Objects * pObjHeap)
+    H264DecoderFrameInfo(H264DecoderFrame * pFrame,  H264_Heap_Objects * /*pObjHeap*/ = nullptr)
```

Comment:
If I understand correctly, after pre-processor we will get: `H264_Heap_Objects * = nullptr`. And I simply don't understand what's that?!

If you don't need 2nd parameter for H264DecoderFrameInfo, then delete it and fix those places where we call it.